app/views/artworks/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+<img class="rounded img-fluid" src="https://images.unsplash.com/photo-1541680670548-88e8cd23c0f4?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=698&q=80" alt="">
+<h1><%= @artwork.name %></h1>
+<h5>Owned by <%= @artwork.user.username %></h5>
+<p><%= @artwork.description %></p>
```

Comment:
Reason why description is not being displayed is because we didn't add it in our seed file, so no worries there.